src/ErrorHandling.c

```diff
@@ -32,6 +32,41 @@ void Ros_ErrorHandling_ErrNo_ToString(int errNo, char* const errMsg, int errMsgS
     }
 }
 
+const char* const Ros_ErrorHandling_Only_ErrNo_ToString(int errNo)
```

Comment:
This seems to duplicate what `Ros_ErrorHandling_ErrNo_ToString(..)` does above. It also appears to return pointers to temporary objects.

Could we use `Ros_ErrorHandling_ErrNo_ToString(..)` instead of adding a new function?

Reply:
Sure! I just had though you wanted it to function similarly to `Ros_ErrorHandling_MotionNotReadyCode_ToString(..)` and return a temporary object which `Ros_ErrorHandling_ErrNo_ToString(..)` does not do. What should the object that should be the destination (the 2nd parameter) that is passed into `Ros_ErrorHandling_ErrNo_ToString(..)` be? Can I create a new buffer specifically for this?

Reply:
> Sure! I just had though you wanted it to function similarly to `Ros_ErrorHandling_MotionNotReadyCode_ToString(..)` and return a temporary object which `Ros_ErrorHandling_ErrNo_ToString(..)` does not do. 

`Ros_ErrorHandling_MotionNotReadyCode_ToString(..)` does not return a pointer-to-temporary. It returns the `const char*` which points to a `static`ally allocated string inside the type support library for `motoros2_interface`.

As follows (from `target_ws\install\include\motoros2_interfaces\msg\detail\motion_ready_enum__struct.h`, generated code):

<details><summary>Click to expand</summary>

```c
// Constants defined in the message

/// Constant 'READY'.
/**
  * Result codes which may be returned by services and actions which attempt to
  * execute motion (or are executing motion).
  *
  * Values other than 1 would indicate an active error or other active inhibition
  * on the Yaskawa controller preventing MotoROS2 from enabling trajectory mode
  * or executing any motion.
  *
  * Applications are encouraged to refer to the named constants generated by
  * rosidl as part of this message instead of using the integer values directly.
 */
enum
{
  motoros2_interfaces__msg__MotionReadyEnum__READY = 1
};

/// Constant 'READY_STR'.
static const char * const motoros2_interfaces__msg__MotionReadyEnum__READY_STR = "Ready";

...
```

</details>

The `static` there changes the lifetime of the strings to essentially `as-long-as-the-program-runs`.

> What should the object that should be the destination (the 2nd parameter) that is passed into `Ros_ErrorHandling_ErrNo_ToString(..)` be? Can I create a new buffer specifically for this?

There are two options I believe:

 1. you just use `Ros_ErrorHandling_ErrNo_ToString(..)` as-is. That would mean you'd have to provide a `char[]` for the `strncpy(..)` to copy into.
 2. we could change `Ros_ErrorHandling_ErrNo_ToString(..)` to also have it return `const char*` to `static`ally allocated strings. That should solve the issue identified here. `Ros_ErrorHandling_ErrNo_ToString(..)` is not currently used, so it would be a relatively minor change without too much fall-out.

I'll push a commit which implements option 2.